OpenEXR/IlmImfFuzzTest/testFuzzDeepScanLines.cpp

```diff
@@ -247,7 +249,7 @@ void readFile(const char filename[])
         file.readPixelSampleCounts(dataWindow.min.y, dataWindow.max.y);
         for (int i = 0; i < dataWindow.max.y - dataWindow.min.y + 1; i++)
         {
-            int y = i + dataWindow.min.y;
+            //int y = i + dataWindow.min.y;
```

Comment:
should probably remove rather than comment, throughout

Reply:
agreed, remove it if it's not used.

Reply:
done